tests/deposit_withdraw_test.ts

```diff
@@ -29,17 +37,19 @@ async function main() {
             [txdeposit]
     );
 
+    let networkId = parseInt(config.deviceId, 10);
     let txwithdraw= new TxWithdraw(
             new BN(0),
             new BN(0),
             new BN(0),
             new BN(1).shln(12),
-            new Address("D91A86B4D8551290655caCED21856eF6E532F2D4")
+            new Address("D91A86B4D8551290655caCED21856eF6E532F2D4"),
```

Comment:
Address need refine the TODO work.
If input address has 0x start, need remove 0x